src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -50,9 +84,17 @@ public override string SelectionSuggestion
         {
             get
             {
-                return RevitDynamoModel.RunEnabled
-                    ? base.SelectionSuggestion
-                    : "Selection is disabled when Dynamo run is disabled.";
+                if (null != revitDynamoModel)
+                {
+                    var hwm = RevitDynamoModel.Workspaces.OfType<HomeWorkspaceModel>().ElementAt(0);
+                    return hwm.RunSettings.RunEnabled
+                        ? base.SelectionSuggestion
+                        : "Selection is disabled when Dynamo run is disabled.";
```

Comment:
`SelectionSuggestion` seems to be user facing, this needs to go into a resource file. Do take note though, the WWL team localizes only resources inside `Revit_2014` and then the localized resources are copied into `Revit_2015` and `Revit_2016`. So it is mandatory that we keep these three branches in-sync, and new strings are added only to `Revit_2014`.